src/Features/Core/Portable/NavigateTo/NavigateToSearcher.cs

```diff
@@ -182,6 +193,15 @@ private async Task SearchCoreAsync(Project project, ImmutableArray<Document> pri
                         {
                             foreach (var result in results)
                             {
+                                // If we're seeing a dupe in another project, then filter it out here.  The results from
+                                // the individual projects will already contain the information about all the projects
+                                // leading to a better condensed view that doesn't look like it contains duplicate info.
+                                lock (seenItems)
+                                {
+                                    if (!seenItems.Add(result))
```

Comment:
Does this mean results from documents linked into multiple multi-targeting projects will only be shown in a single multi-targeting project? (e.g. PooledHashSet in Roslyn)

Reply:
yes.  that's what it means :)